alerts/alerts-dev.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: "nais.io/v1alpha1"
```

Comment:
Vi har skrudd av våre dev-alerts - det endte bare opp med at vi ikke brydde oss noe særlig om dem, og det gjør at vi må vedlikeholde flere (nesten helt like) config-filer. Skal dere faktisk ha alerts i dev?

Reply:
Jeg endte opp med å bare bruke prod-alerten, og teste den i test-kanalen før jeg satt den i prod.

Reply:
Ja lurte litt på det samme.
Om man får noen av varslene i testkanalen i Slack så kan jo det gi en indikasjon på at noe er galt før man eventuelt legger noe ut i prod...
Det skader jo ikke å ha begge to. Men er jo en mulighet for at vi bare fjerner dev alerten hvis det ikke gir noe verdi og bare er støy.